### PR TITLE
Move reproducibility to raw.

### PR DESCRIPTION
**Pull request by layik**

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -41,7 +41,8 @@ Suggests:
     stringr,
     testthat,
     tidyr,
-    pkgdown
+    pkgdown,
+    geojsonsf
```
Comment (Robinlovelace) [inline DESCRIPTION]: I don't think this is needed if we only use the package for reading one file in `data-raw`. Also, the -s of using another package, when `sf` works fine (if a bit slower, although it's a tiny dataset) outweigh +s of the speed of `geojsonsf` IMO.

**Comment by layik** [comment]
A bit messy from me, should have started this from "base"/master and not include the other two branches. On the +s side, it means once we accept this all three branches have been merged into master.

File: R/data.R [hunk 2]
```diff
@@ -29,31 +29,13 @@ NULL
...
-#' for(i in 1:length(r)) {
-#'   dr[i] = sub("http://data.dft.gov.uk.s3.amazonaws.com/road-accidents-safety-data/",
-#'               "", URLdecode(r[i]))
-#'   dr[i] = sub("http://data.dft.gov.uk/road-accidents-safety-data/",
-#'               "", dr[i])
-#' }
-#' file_names = setNames(as.list(file_names), file_names)
-#' usethis::use_data(file_names)
+#'  length(file_names)
+#'  file_names$dftRoadSafetyData_Vehicles_2017.zip
```
Comment (Robinlovelace) [inline R/data.R]: :+1:

File: R/data.R [hunk 3]
```diff
@@ -63,15 +45,13 @@ NULL
...
 #' \donttest{
-#' # Obtained with:
-#' dl_stats19(year = 2017, type = "Accide")
-#' accidents_2017_raw = read_accidents(year = 2017)
-#' set.seed(350)
-#' sel = sample(nrow(accidents_2017_raw), 3)
-#' accidents_sample_raw = accidents_2017_raw[sel, ]
-#' accidents_sample = format_accidents(accidents_sample_raw)
+#' nrow(accidents_sample_raw)
+#' accidents_sample_raw
```
Comment (Robinlovelace) [inline R/data.R]: :+1: although I don't think that it was doing any harm before.

File: R/data.R [hunk 4]
```diff
@@ -82,15 +62,13 @@ NULL
...
 #' \donttest{
-#' # Obtained with:
-#' dl_stats19(year = 2017, type = "cas")
-#' casualties_2017_raw = read_casualties(year = 2017)
-#' set.seed(350)
-#' sel = sample(nrow(casualties_2017_raw), 3)
-#' casualties_sample_raw = casualties_2017_raw[sel, ]
-#' casualties_sample = format_casualties(casualties_sample_raw)
+#' nrow(casualties_sample_raw)
+#' casualties_sample_raw
```
Comment (Robinlovelace) [inline R/data.R]: :+1:

**Comment by layik** [comment]
> remove this new dependency though

geojsonsf? Package would break.

> move the whole example into data-raw, as per another reviewer comment.

Have I not?

File: R/data.R [hunk 5]
```diff
@@ -101,15 +79,13 @@ NULL
...
 #' \donttest{
-#' # Obtained with:
-#' dl_stats19(year = 2017, type = "veh")
-#' vehicles_2017_raw = read_vehicles(year = 2017)
-#' set.seed(350)
-#' sel = sample(nrow(vehicles_2017_raw), 3)
-#' vehicles_sample_raw = vehicles_2017_raw[sel, ]
-#' vehicles_sample = format_vehicles(vehicles_sample_raw)
+#' nrow(vehicles_sample_raw)
+#' vehicles_sample_raw
```
Comment (Robinlovelace) [inline R/data.R]: :+1:

File: R/data.R [hunk 6]
```diff
@@ -122,16 +98,12 @@ NULL
...
-#' # Obtained with:
-#' library(sf)
-#' u = paste0("https://opendata.arcgis.com/",
-#'   "datasets/3e5a096a8c7c456fb6d3164a3f44b005_3.kml"
-#'   )
-#' police_boundaries_wgs = read_sf(u)
-#' police_boundaries = st_transform(police_boundaries_wgs, 27700)
-#' police_boundaries = police_boundaries[c("pfa16cd", "pfa16nm")]
+#' nrow(police_boundaries)
+#' police_boundaries[police_boundaries$pfa16nm == "West Yorkshire", ]
```
Comment (Robinlovelace) [inline R/data.R]: Big :+1:

**Comment by layik** [comment]
I see what you mean. I think the reviewer states

> final package with proper documentation as to what the data are and a link to where they were obtained from

I take it that also means an example as to how to use the data, e.g, the geometry for a particular police force boundary, etc.

File: DESCRIPTION (hunk 1 above)
Comment (layik) [reply DESCRIPTION]: Could you please use `sf`, my Googles were not helping me, that is why I opted for the geojson of it.

Comment (layik) [reply DESCRIPTION]: But the reviewer is right, as it stands master does not work and returns two empty list items (Name and Description).

Comment (Robinlovelace) [reply DESCRIPTION]:
```
st_read("police_boundaries.geojson")
```

Comment (layik) [reply DESCRIPTION]: I see :+1: sorry was miles away.

Comment (layik) [reply DESCRIPTION]: Done.

Comment (Robinlovelace) [reply DESCRIPTION]: Awesome. Many thanks!

**Comment by layik** [comment]
Awaiting "approval" or further comments @Robinlovelace.

**Comment by Robinlovelace** [comment]
Thanks @layik, great work!

**Pull request merged**